src/CampaignsAdmin/report/pdf/PrintToPdf.tsx

```diff
@@ -12,7 +12,7 @@ export const PrintToPdf = ({campaign}: PrintToPdfProps) => {
 
   return (
     <>
-      <div style={{ display: 'none'}}>
+      <div style={{ position:'absolute', visibility:'hidden' }}>
```

Comment:
Why are these styles set inline if you use CSS classes for everything else?  Limitation of the PDF library?